### DISCUSSION TITLE
/accounts/signup/ page says "Sign Up Closed" even when "Enable Registration" is ticked.

### DISCUSSION THREAD

**Discussion by Southpaw1496** [inventree/InvenTree#6152] [Q&A]
I'm setting up Inventree, and I want to create a few user accounts without needing to use SSO. I couldn't find a setting to manually create accounts for other users, so I ticked "Enable Registration", but if I open a private window and click "Sign Up" on the log in page, it says that sign up is closed. Are there any other things I need to change in order to enable sign up?

**Comment by SchrodingersGat** [comment]
Do you have your email information correctly configured?

**Comment by Southpaw1496** [comment]
I haven't set up the SMTP system yet. Is that a requirement?

**Reply by SchrodingersGat** [reply]
I'm not at my computer to check but I believe so
